perun-core/src/test/java/cz/metacentrum/perun/core/entry/ModulesUtilsEntryIntegrationTest.java

```diff
@@ -40,6 +40,8 @@
 import cz.metacentrum.perun.core.api.exceptions.WrongAttributeValueException;
 import cz.metacentrum.perun.core.bl.ModulesUtilsBl;
 import cz.metacentrum.perun.core.impl.PerunSessionImpl;
+import sun.awt.image.ImageWatched;
```

Comment:
Probably bad autoimport by IDEA and it shouldn't be here.

Reply:
Removed.